### PR TITLE
Visualization v1

### PR DESCRIPTION
**Pull request by diegotc86**
Feature #2: Measures visualization

This pull request include the following:
- Display user measures per day on a table. To change the user set the find parameter on the get route "/view_measures"
- The table includes the calculated BMI for each day
- The table shows the increment and decrement (variation) for height, weight and BMI
- The weight variation cell has a red/green background color according if it is going toward or against the ideal weight (calculated base on the height and gender)
- Cell coloring change according if Ideal Weight or Set Goal is selected
- Integration with login view

### CODE REVIEW

File: routes.rb [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
 end
 
-get '/user' do
-  erb :user
+# get '/user' do
+#   erb :user
+# end
+
+get "/view_measures" do
+  @user_data = User::find("diego@mail.com")
```
Comment (KattyaCuevas) [inline routes.rb]: Why do you use this syntax instead of `User.find("diego@mail.com")`?

File: views/view_measures.erb [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+          <th>Height +/-</th>
+          <th>Weight</th>
+          <th>Weight +/-</th>
+          <th>BMI</th>
+          <th>BMI +/-</th>
+          <th>Ideal Weight</th>
+        </tr>
+      </thead>
+      <tbody>
+        <% last_height = 0 %>
```
Comment (KattyaCuevas) [inline views/view_measures.erb]: The logic data shouldn't be in the view. Maybe you should move the logic to the model.

File: routes.rb (hunk 1 above)
Comment (diegotc86) [reply routes.rb]: We thought that was the way to call a class method. We will use instead `User.find("diego@mail.com")` as clearly is the right way. Thanks.

File: views/view_measures.erb (hunk 2 above)
Comment (diegotc86) [reply views/view_measures.erb]: I will work on this on the next iteration.

**Pull request merged**